<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXExitDialog</class>
 <widget class="QDialog" name="CXExitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CXExitDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="0" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>873</width>
       <height>248</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="CXTouchButton" name="mTurnOffButton">
     <property name="text">
      <string>Выключить УЧПУ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>873</width>
       <height>248</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="CXTouchButton" name="mCancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>365</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="CXTouchButton" name="mExitButton">
     <property name="text">
      <string>Выход без выключения</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>364</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CXTouchButton</class>
   <extends>QPushButton</extends>
   <header>CXTouchButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
